src/views/evaluation/Evaluation.js

```diff
@@ -0,0 +1,122 @@
+const NAV = function (option = {}){
+    return `
+    <ul class="header headerT">
+        ${logoButton(option.Logo)}
+        ${constructNav(option)}
+    </ul>`
+}
+
+const logoButton = function(option = {}){
+    return `
+    <li class="header-elements">
+        <img class= "main-logo" src="${option}">
+    </li>`
+}
+
+const buttonNav = function(option={}, active="" , specialPlacement = ""){
+    return `
+    <li class="header-elements ${specialPlacement}">
+        <a class = "header-elements-link header-elements-linkT ${active} " href="${option.Link}"> ${option.Text}</a>
+    </li>
+    `
+}
+
+
+const constructNav = function(option = {}){
+    const currentPage = "Evaluationv2.html", result =[], specialPlacement = "Loginv2.html";
+    for(var i=0;i<option.Link.length;i++){
+        if(option.Link[i] === currentPage){
+            result.push(buttonNav({Link: option.Link[i], Text: option.Text[i]}, "active-page"))
+        }else{ 
+            if(option.Link[i] === specialPlacement){
+                result.push(buttonNav({Link: option.Link[i], Text: option.Text[i]}, "", "right-button"))
```

Comment:
Related tot this part of code. The thing I tried to do there would be something like this: 
I have as a local variable "currentPage" which has the link of the current page. This goes into the if statement and check if the data for the actual button is also the data of the actuall page.  If this this true then it will add at the button class the "active-page" tag and makes it be the active button making it underlined. Then I notated as "special-button" the ones that should be "float:right" and I go for this on the else and check wether the button is Logout or not, and if it is the it is added to the class. 
I can think a simpler way of changing but I don't think the active page thing can be changed